### PR TITLE
fix gpu tests

### PR DESCRIPTION
**Pull request by bwanglzu**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -269,3 +270,34 @@ The changes of each release is tracked by this file.
  - [[```43d62f06```](https://github.com/jina-ai/finetuner/commit/43d62f068fdf80ca0a1a4d9f86ec24804f7f6aca)] __-__ __readme__: update logo (*Han Xiao*)
  - [[```489014ee```](https://github.com/jina-ai/finetuner/commit/489014ee4e12e6c3cc697d7c4da4129dd600ccdb)] __-__ __version__: the next version will be 0.1.4 (*Jina Dev Bot*)
 
+<a name=release-note-0-1-5></a>
```
Comment (tadejsv) [inline CHANGELOG.md:273]: ?

Comment (bwanglzu) [reply CHANGELOG.md:273]: no idea, i think it's somethign related to jina bot

Comment (tadejsv) [reply CHANGELOG.md:273]: I see - I think this came when you merged with my branch for some reason. It's strange because this is alreday part of my branch

File: tests/unit/tuner/keras/test_gpu.py [hunk 2]
```diff
@@ -1,57 +1,43 @@
...
+from finetuner.embedding import embed
+from finetuner.toydata import generate_fashion
+
+all_test_losses = ['SiameseLoss', 'TripletLoss']
+
+
+@pytest.mark.gpu
+@pytest.mark.parametrize('loss', all_test_losses)
+def test_gpu_keras(generate_random_data, loss, tf_gpu_config):
+    data = generate_random_data(4, 4)
```
Comment (tadejsv) [inline tests/unit/tuner/keras/test_gpu.py:15]: Can you change this as well

**Comment by bwanglzu** [comment]
i'll create a seperate PR to test edge cases for loss functions and fix paddle there.

**Review by tadejsv — APPROVED** [review]
